vendor/github.com/onsi/ginkgo/internal/remote/BUILD.bazel

```diff
@@ -0,0 +1,51 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
let's skip those bazel files. k/k has removed bazel building, we should remove it as well.

Reply:
opened an issue to track: https://github.com/kubernetes-sigs/cloud-provider-azure/issues/557